src/Sulu/Bundle/ContentBundle/Admin/ContentContentNavigationProvider.php

```diff
@@ -82,7 +78,7 @@ public function getNavigationItems(array $options = array())
             $permissions->setComponentOptions(
                 array(
                     'display' => 'form',
-                    'type' => Structure::class,
+                    'type' => PageDocument::class,
```

Comment:
this isn't what you want (e.g. `HomeDocument` also should be secured). You could use `BasePageDocument`, but that is not what that class is meant to be used for.

I would prefer to create a `PageBehavior` that is implemented by the `BasePageDocument`. (this behavior already exists, but is not implemented).

What methods do you require from the document?